Updated and fixed READMEs ++ (#70)
fredrikhgrelland/vagrant-hashistack-template: dev/vagrant/conf/nomad/README.md

# Nomad configuration 

> You may add any hcl-files to this directory in order to change or override the configuration.

Reviewer: The same comment as above about json config files
```suggestion
You may add any hcl-files to this directory in order to change or override the configuration.
```

Any valid [nomad configuration](https://www.nomadproject.io/docs/configuration#general-parameters) added to this directory will append to the configuration, in lexical order. 

> It means that by adding the file `00-override.hcl` will be read first and the file `99-override.hcl` you will ensure it will be read last.

Reviewer 2: ```suggestion
It means that by prefixing a filename with `00` (e.g. `00-override.hcl`), it will be read first and using `99` (e.g. `99-override.hcl`) it will be read last.
```

}
```

> A example file is provided in [template_example/dev/vagrant/conf/nomad](https://github.com/fredrikhgrelland/vagrant-hashistack-template/tree/master/template_example/dev/vagrant/conf/nomad).

Reviewer: ```suggestion
An example file is provided in [template_example/dev/vagrant/conf/nomad](https://github.com/fredrikhgrelland/vagrant-hashistack-template/tree/master/template_example/dev/vagrant/conf/nomad).
```
